src/backend/tests/conftest.py

```diff
@@ -230,6 +230,7 @@ async def task_event(db, project, tasks, admin_user):
         new_event = TaskEventIn(
             task_id=task.id,
             user_id=user.id,
+            username=user.username,
```

Comment:
I'm surprised this isn't calculated in the SQL, using the user id?

If you insert both the user id and username, then it opens the potential for inconsistency: inserting a username that does not match the user id

Reply:
Ah you are right , I got what you meant to say. I was trying to solve the test case failure issue, but it seems to have github action issue?